packages/utility/agent.ts

```diff
@@ -1,75 +1,50 @@
-import { NewApplicant, Opportunity, Project, User as UserType, Idea, Like, Save } from "@devlaunchers/models";
```

Comment:
can we revert the changes that removed comments from this file `packages/utility/agent.ts`? why have you removed it in the first place if I may ask?